rules/utils.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+function bindings(node) {
```

Comment:
`bindings()` is an ambiguously-named function in a file with a generic name `utils.js`, but it's specific to `delegated-events`. Perhaps `bindings` could be better named?

Reply:
1e32d564ed3c7974338a84ec2533729023b7e3a1